modules/cluster-logging-deploy-cli.adoc

```diff
@@ -313,40 +222,40 @@ spec:
     elasticsearch:
       nodeCount: 3 <5>
       storage:
-        storageClassName: "<storage-class-name>" <6>
+        storageClassName: <storage_class_name> <6>
         size: 200G
       resources: <7>
         limits:
-          memory: "16Gi"
+          memory: 16Gi
         requests:
-          memory: "16Gi"
+          memory: 16Gi
       proxy: <8>
         resources:
           limits:
             memory: 256Mi
           requests:
              memory: 256Mi
-      redundancyPolicy: "SingleRedundancy"
+      redundancyPolicy: SingleRedundancy
   visualization:
-    type: "kibana"  <9>
+    type: kibana  <9>
     kibana:
       replicas: 1
   collection:
     logs:
-      type: "fluentd"  <10>
+      type: fluentd  <10>
       fluentd: {}
 ----
 <1> The name must be `instance`.
-<2> The OpenShift Logging management state. In some cases, if you change the OpenShift Logging defaults, you must set this to `Unmanaged`.
-However, an unmanaged deployment does not receive updates until OpenShift Logging is placed back into a managed state. Placing a deployment back into a managed state might revert any modifications you made.
+<2> The {logging} management state. In some cases, if you change the {logging} defaults, you must set this to `Unmanaged`.
```

Comment:
here, too, the line break is working for the preview, but it could probably be deleted